### DISCUSSION TITLE
java.lang.UnsupportedOperationException: Empty collection can't be reduced.

### DISCUSSION THREAD

**Discussion by wizenolifahub** [Blad3Mak3r/Killjoy#40] [Q&A] [answered]
I am very new to these issues and want to improve myself, can you help me with this error? (I'm sorry for my bad english)

**Comment by Blad3Mak3r** [comment, marked as answer]
Heyy welcome!!

I think this is not the best place to ask questions about things that are not related to Killjoy, as this section is intended to offer support to users who use the bot on their Discord servers.

A good place for your Java questions is Stackoverflow where you can find answers to your questions. 

https://stackoverflow.com/questions/tagged/java
